assignment-2/7s.md

```diff
@@ -1,2 +1,171 @@
+- Other competitors in the market like Grooveshark have been shut down as part of legal settlements, because strategies that did not compensate the music production, Spotify even having some artist criticism, has manage to hold some balance between their activities and the music production industry.
+
+##### Structure
+
+> Structure is about how the organization is arranged or configured. Many organizations take a
+> functional structure, in which different branches of the organization specialize in particular
+> functions, such as manufacturing, sales, and service. Another common form is a divisional
+> structure, in which divisions are arrayed according to geographical regions, products, or some
+> other factor, and each division contains a full complement of functional elements (e.g., sales,
+> service, R&D).
+
+Spotify headquarters resides in London and its handles, research and development departments are in Stockholm.
+Like many other matrix management organizations, Spotify subdivides the work among people with similar skills, and they are pooled for work assignments, resulting in more than one manager with their own responsibilities and competencies. All this managers are from that point, reporting hierarchical button up to the CTO and finally to the CEO.
+
+##### Systems
+
+> Systems refers to the rules and regulations, standards, and processes for getting things done and managing the activity of the organization, both formal and informal.
+
+How things get done? 
+according to Spotify [12]: 
+- Spotify has a culture based on agile principles, all engineerings happens in squads, they try to keep those squads loosely coupled, tightly aligned.
+- They like cross-pollination instead of 'standarization', in order to spread knowledge, trends and tools between squads. It does not break the principal of 'issolated missions' because cross-pollination is used to spread the uses of tools like github for example. 
+- Their code is shared on an internal open source model. 
+- Squads make small and frequent releases as a routine, which is enable due decoupling their services. 
+- Their self-service model pretend minimizes the need for handoffs. 
+- They uses release trains and feature toggles aiming get things into production early and often.
+- They claim that their culture is based also on the people, therefore the focus on motivation, community and trust rather than structure and control.
+- Product development is based on Lean. They release their new features to a small group of customers limiting the 'blast radius' of a possible fail and it allows them to conduce a lot of small experiments. They report that it produces a lot of waste but this can be seen from a Lean perspective.
+They have something called 'core metrics data'. The data is feed dynamically due constant experimentation.
+
+##### Staffing
+
```

Comment:
Can you tell more about this. How does Spotify recruit their employees, how do they select their leaders, what are they looking for in an employee? How do they know that somebody will perform? What do you think about their staffing strategy? Etc.